adopt-a-pup/web-app/src/Components/AnimalCreateForm.tsx

```diff
@@ -541,4 +565,41 @@ export default class AnimalCreateForm
         }
         return null;
     }
+
+    private renderPhotoPickerModal() {
+        // TODO: load photo list from somewhere else
+        const photos = [
```

Comment:
In a real app, we would probably load this list with an AJAX call, but in this case, I think it's enough if we just load a hardcoded photo list.

So.. I'm moving this array to a separate file.